tests/integration/data_sources_and_expectations/data_sources/test_databricks.py

```diff
@@ -0,0 +1,164 @@
+    type supported by Databricks.
+
+    https://docs.databricks.com/en/sql/language-manual/sql-ref-datatypes.html
+    """
+
+    BOOL_COL_NAME = "my_bool"
+    DATE_COL_NAME = "my_date"
+    NUMERIC_COL_NAME = "my_number"
+    STRING_COL_NAME = "my_string"
+
+    DATA_FRAME = pd.DataFrame(
+        {
+            BOOL_COL_NAME: [True, False, True, False],
+            DATE_COL_NAME: [
+                datetime(2021, 1, 1, tzinfo=timezone.utc).date(),
+                datetime(2021, 1, 2, tzinfo=timezone.utc).date(),
+                datetime(2021, 1, 3, tzinfo=timezone.utc).date(),
+                datetime(2021, 1, 4, tzinfo=timezone.utc).date(),
+            ],
+            NUMERIC_COL_NAME: [1, 2, 3, 4],
+            STRING_COL_NAME: ["a", "b", "c", "d"],
+        }
+    )
+
+    @pytest.mark.parametrize(
+        "column_type",
+        [
+            sqltypes.SMALLINT,
+            sqltypes.INT,
+            sqltypes.BIGINT,
+            sqltypes.DECIMAL,
+            sqltypes.FLOAT,
+            sqltypes.REAL,
+        ],
+    )
+    def test_number(self, column_type: sqltypes.TypeEngine):
+        batch_setup = DatabricksBatchTestSetup(
+            config=DatabricksDatasourceTestConfig(
+                column_types={self.NUMERIC_COL_NAME: column_type}
+            ),
+            data=self.DATA_FRAME,
+            extra_data={},
+        )
+        with batch_setup.batch_test_context() as batch:
+            result = batch.validate(
+                expect=ExpectColumnSumToBeBetween(
+                    column=self.NUMERIC_COL_NAME,
+                    min_value=9,
+                    max_value=11,
+                )
+            )
+        assert result.success
+
+    def test_varchar(self):
+        column_type = sqltypes.String
+        batch_setup = DatabricksBatchTestSetup(
+            config=DatabricksDatasourceTestConfig(column_types={self.STRING_COL_NAME: column_type}),
+            data=self.DATA_FRAME,
+            extra_data={},
+        )
+        with batch_setup.batch_test_context() as batch:
+            result = batch.validate(
+                expect=ExpectColumnDistinctValuesToContainSet(
+                    column=self.STRING_COL_NAME,
+                    value_set=[
+                        "a",
+                        "b",
+                    ],
+                )
+            )
+        assert result.success
+
+    def test_boolean(self):
+        column_type = sqltypes.BOOLEAN
+        batch_setup = DatabricksBatchTestSetup(
+            config=DatabricksDatasourceTestConfig(column_types={self.BOOL_COL_NAME: column_type}),
+            data=self.DATA_FRAME,
+            extra_data={},
+        )
+        with batch_setup.batch_test_context() as batch:
+            result = batch.validate(
+                expect=ExpectColumnValuesToBeInSet(
+                    column=self.BOOL_COL_NAME, value_set=[True, False]
+                )
+            )
+        assert result.success
+
+    @pytest.mark.xfail(
```

Comment:
is there a ticket for this?

Reply:
https://greatexpectations.atlassian.net/browse/CORE-692